### PR TITLE
Issue #8252: Update text on Nightly What's New page

### PR DESCRIPTION
**Pull request by pascalchevrel**
## Description
Minor updates to the what's new page

## Issue / Bugzilla link
Issue #8252 

## Testing
* The IRC paragraph should be removed
* The sentence "Every 6 to 8 weeks…" should now be "Every 4 to 5 weeks…"

### CODE REVIEW

File: bedrock/firefox/templates/firefox/nightly_whatsnew.html [hunk 1]
```diff
@@ -39,7 +39,7 @@
           <h2>{{ self.page_title() }}</h2>
           <p class="tagline">
           {% trans %}
-            Every 6 to 8 weeks, a new major version of Firefox is released and as a result, the Nightly version
+            Every 4 to 5 weeks, a new major version of Firefox is released and as a result, the Nightly version
```
Comment (craigcook) [inline bedrock/firefox/templates/firefox/nightly_whatsnew.html]: This page is localized so we should use `l10n_has_tag` for this string change.

Comment (pascalchevrel) [reply bedrock/firefox/templates/firefox/nightly_whatsnew.html]: done

**Comment by pascalchevrel** [comment]
@peiying2 FYI, this PR contains a string change behind an l10n tag, the updated string should be added to the lang file.

File: bedrock/firefox/templates/firefox/nightly_whatsnew.html [hunk 2]
```diff
@@ -74,12 +77,6 @@ <h2>{{ self.page_title() }}</h2>
               karma if you add the <em>nightly-community</em> keyword to your bug reports)!
             {% endtrans %}
           </p>
-          <p>
-            {% trans irc='ircs://irc.mozilla.org/nightly' %}
-              You can also interact with the Nightly community on IRC in the <a href="%(irc)s">#nightly</a> channel.
-              Feel free to ask questions and talk about features, regressions or crashes you may be experiencing.
-            {% endtrans %}
-          </p>
```
Comment (jens1o) [inline bedrock/firefox/templates/firefox/nightly_whatsnew.html]: I wonder why `chat.mozilla.org` wasn't picked up here?

Comment (pascalchevrel) [reply bedrock/firefox/templates/firefox/nightly_whatsnew.html]: Because it didn't exist at the time and we had not moved to Matrix.

Comment (jens1o) [reply bedrock/firefox/templates/firefox/nightly_whatsnew.html]: Should I open up a new bug for it?

Comment (pascalchevrel) [reply bedrock/firefox/templates/firefox/nightly_whatsnew.html]: Yes please (and CC me), thanks!

Comment (jens1o) [reply bedrock/firefox/templates/firefox/nightly_whatsnew.html]: Did that: https://github.com/mozilla/bedrock/issues/8688

Sorry for the long wait...

**Pull request merged**